### PR TITLE
Delete data

### PR DESCRIPTION
**Pull request by noramullen1**

### CODE REVIEW

File: content/v2.0/write-data/delete-data.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+weight: 104
+v2.0/tags: [delete]
+related:
+  - /v2.0/reference/syntax/delete-predicate/
+  - /v2.0/reference/cli/influx/delete/
+---
+
+## Delete data in the InfluxDB UI
+
+You can only delete data from buckets you've created. You cannot delete data from system buckets.
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: Delete data f...

File: content/v2.0/write-data/delete-data.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+1. Click **Load Data** in the navigation bar.
+
+    {{< nav-icon "load data" >}}
+
+2. Select **Buckets**.
+3. Next to the bucket with data you want to delete, click **Delete Data by Filter**.
+4. In the **Delete Data** window that appears:
+  - Select a **Target Bucket** to delete data from.
+  - Enter a **Time Range** to search.
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: time range to delete data from?

File: content/v2.0/write-data/delete-data.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  - Click **+ Add Filter** to filter by tag key and value pair.
+  - Operator other than equals?
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: - Select operator other than equals (remove question mark?)

File: content/v2.0/write-data/delete-data.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  - Check the box next to **I understand that this cannot be undone**.
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: I'd prob use Google style for check boxes: Select **I understand....

File: content/v2.0/write-data/delete-data.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+### Delete data from the Data Explorer
+
+1. Click the **Data Explorer** icon in the sidebar.
+
+    {{< nav-icon "data-explorer" >}}
+
+2. Click **Delete Data** in the top navigation bar.
+3. In the **Delete Data** window that appears:
+  - Select a **Target Bucket** to delete data from.
+  - Enter a **Time Range** to search.
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: time range to delete data from?

File: content/v2.0/write-data/delete-data.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  - Click **+ Add Filter** to filter by tag key and value pair.
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: have seen "key-value pair" in 1.x docs, shall we use that standard?....one for our internal "word list"

File: content/v2.0/write-data/delete-data.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  - Check the box next to **I understand that this cannot be undone**.
+4. Click **Confirm Delete** to delete the selected data.
+
+## Delete data using the influx CLI
+Use the [`influx delete` command](/v2.0/reference/cli/influx/delete/) to delete points from InfluxDB.
+Specify your organization, bucket, and authentication token.
+Define the time range to delete data from with the `--start` and `--stop` flags.
+Specify which points to delete using the `--predicate` or `-p` flag and
+[Delete predicate syntax](/v2.0/reference/syntax/delete-predicate/).
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: maybe number these steps?

File: content/v2.0/write-data/delete-data.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+timestamps between the specified `--start` and `--stop` times in the specified bucket.
+{{% /warn %}}
+
+## Delete data using the API
+Use the InfluxDB API `/delete` endpoint to delete points from InfluxDB.
+Include your organization and bucket as query parameters in the request URL.
+Use the `Authorization` header to provide your InfluxDB authentication token.
+In your request payload, define the time range to delete data from with `start` and `stop`.
+Specify which points to delete using the `predicate` and
+[Delete predicate syntax](/v2.0/reference/syntax/delete-predicate/).
```
Comment (kelseiv) [inline content/v2.0/write-data/delete-data.md]: maybe number?

File: content/v2.0/write-data/delete-data.md (hunk 3 above)
Comment (noramullen1) [reply content/v2.0/write-data/delete-data.md]: Whoops, was supposed to delete that!

**Comment by kelseiv** [comment]
Closing this PR - replaced by https://github.com/influxdata/docs-v2/pull/1642/files

**Pull request closed without merging**